### PR TITLE
[Enhancement] create subpackage for HTML guides

### PR DESCRIPTION
**Pull request by redhatrises**
- Creates a subpackage for HTML-generated guides

### CODE REVIEW

**Comment by mpreisler** [comment]
It would be great if the upstream spec matched how the Fedora downstream spec does things. We have plan to sync them eventually.

See http://pkgs.fedoraproject.org/cgit/scap-security-guide.git/tree/scap-security-guide.spec#n83

**Comment by redhatrises** [comment]
@mpreisler merged in some of the Fedora spec changes. 

It would be nice to have a single spec file (if possible) for Fedora and RHEL. I could make the spec file exactly like the Fedora one if so desired.

**Comment by isimluk** [comment]
This is definitely improvement. So, I would like to merge it.

However, there is one difference that is worth fixing in this PR. Fedora's sub package is called `-doc' while here we propose `-docs`. I think it makes sense to have `doc` everywhere.


Convergence of specfile is useful but hardish to achieve in real world. Specfiles in downstreams and upstream serve different purpose. Downstream specfile ensures that rules of particular distribution are followed and ensure certain level of quality. Upstream specfile is more for innovators who want to use latest greatest SSG in their environment.

The way to approach convergence is the following: all the specfiles shall make a use of upstream's `make install`. For this we need to make sure that `make install` is something like `greatest common divisor`... i.e. all the specfiles are installing DataStreams for rhel5/6/7. So the `make install` should be used to do this task.

**Comment by mpreisler** [comment]
Agreed with @isimluk . Isee the subpackage name matching as a prerequisite for the merge.

**Comment by redhatrises** [comment]
@mpreisler @isimluk 3f72bbf in this PR uses `-doc` instead of `-docs` as requested.

File: scap-security-guide.spec.in [hunk 1]
```diff
@@ -77,31 +85,31 @@ rm %{buildroot}%{_datadir}/xml/scap/ssg/content/*-cpe-dictionary.xml
...
+%if 0%{?fedora}
+cp -a Fedora/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+cp -a Chromium/output/*-guide-*.html %{buildroot}/%{_docdir}/%{name}/guides
+#cp -a Webmin/output/*-guide-*.html %{buildroot}/%{_defaultdocdir}/%{name}/guides
+%endif
 
 %files
 %{_datadir}/xml/scap
 %{_datadir}/%{name}/remediation_functions
+%{_datadir}/%{name}
```
Comment (mpreisler) [inline scap-security-guide.spec.in]: The previous 2 lines cause:
```
warning: File listed twice: /usr/share/scap-security-guide/remediation_functions
```

Drop the `%{_datadir}/%{name}/remediation_functions` line. This is what the Fedora spec does.

**Comment by redhatrises** [comment]
@mpreisler done.

**Comment by mpreisler** [comment]
OK, ACK.

**Pull request merged**